docs/syntax/wireframe.md

```
@@ -0,0 +1,505 @@
+      end
+    end
+  end
+```
+
+## Declaration & Canvas Size Presets
+
+Every wireframe diagram starts with the `wireframe-beta` keyword. You can optionally specify a title enclosed in quotes and a canvas preset size using `size=<preset>`.
+
+### Syntax
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)